src/test/java/hudson/plugins/git/browser/casc/GitLabConfiguratorTest.java

```diff
@@ -0,0 +1,136 @@
+package hudson.plugins.git.browser.casc;
+
+import hudson.plugins.git.browser.GitLab;
+import io.jenkins.plugins.casc.ConfigurationContext;
+import io.jenkins.plugins.casc.ConfiguratorRegistry;
+import io.jenkins.plugins.casc.model.Mapping;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
```

Comment:
Import for later assertion use
```suggestion
import static org.junit.Assert.assertNull;
import static org.junit.Assert.assertThat;
```